scripts/testing/stress-mem.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Stress testing in regard to memory consumption - could cause OOM (but should not).
+# Creates btp-operator and numerous service instances and service bindings in current context.
+#
+# The script has the following arguments:
+#                       - number of service instances and service bindings to be created - default 500
+#                       - number of seconds to let them be before btp-operator is deleted - default 60
+# Example:
+#           ./stress-mem.sh 100 30
+
+N=${1-500}
+YAML_DIR=./scripts/testing/yaml
+LIFE_SPAN=${2-60}
+
+echo -e "\n---Installing BTP operator"
+kubectl apply -f - <<EOF
```

Comment:
we can apply `e2e-test-btpoperator.yaml` and then label it with `kubectl label`, like this: `kubectl label -f e2e-test-btpoperator.yaml force-delete=true`

Reply:
In the next commit.